ipld/cid/Cargo.toml

```diff
@@ -19,6 +19,7 @@ multihash = { version = "0.16", default-features = false, features = [
     "blake2b",
     "derive",
     "serde-codec",
+    "identity",
```

Comment:
Surprisingly, it is. We didn't notice before because it's enabled by `forest_libp2p`. But if you try to build the `actor_interface` without `forest_libp2p` in your workspace, it fails.

Reply:
So it's not related to the original purpose of this PR but it is required to build and test the changes in the `interpreter`.

Reply:
Gotcha!